lmcache/cache_engine.py

```diff
@@ -194,14 +195,17 @@ def _make_chunks_skip_existing(
                                          num_skip_prefix_chunk)
         # With num_skip_chunks, the following is relative to
         # the new start after skip.
-        num_tokens: int = self._num_tokens_in_kv(kv_tensors, fmt)
 
         start_token_idx = None
         start_chunk_idx = 0
-        for chunk_hash, idx in zip(chunk_hashes,
-                                   range(0, num_tokens, self.chunk_size)):
-            if not self.engine_.contains(self._make_key(chunk_hash, fmt)):
-                start_token_idx = idx
+        keys = []
+        for chunk_hash in chunk_hashes:
+            keys.append(self._make_key(chunk_hash, fmt))
+        anws = self.engine_.batched_contains(keys,
+                                             total_size=_get_size(kv_tensors))
+        for anw in anws:
+            if not anw:
```

Comment:
The loop is easy to understand. But what does `anw` and `anws` mean? 

Note that the code might be changed by other people in the future, it's better to have a clear variable name, thanks!